CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## [0.12.2] - TBD
+
+### Added
+- AWS KMS server Keymanager plugin (#2066)
```

Comment:
This line and the next line should probably be worded similarly since they both talk about the introduction of new plugins.